### PR TITLE
Throw exception when other permissions are mixed with wildcard

### LINKED ISSUE #956: Prevent conflicting trust and permission configurations in contracts
According to [this issue](https://github.com/neo-project/neo/issues/2890), neow3j allows contract manifests that hold conflicting permission and trust configurations.

E.g., output that looks like this:
```json
"permissions" : [
            {
               "contract" : "0xfe924b7cfe89ddd271abaf7210a80a7e11178758",
               "methods" : "*"
            },
            {
               "contract" : "*",
               "methods" : "*"
            }
         ]
```
Or this
```
   "trusts" : [
      "0xfe924b7cfe89ddd271abaf7210a80a7e11178758",
      "*"
   ]
```

### PR DESCRIPTION
**Pull request by csmuller**
Closes #956

### CODE REVIEW

File: compiler/src/test/java/io/neow3j/compiler/PlaceholderSubstitutionTest.java [hunk 1]
```diff
@@ -33,16 +33,16 @@ public static void main() {
     @Test
     public void testMultiAnnotationReplacement() throws IOException {
         Map<String, String> replaceMap = new HashMap<>();
-        replaceMap.put("INVALID_TRUST_HASH1", "*");
-        replaceMap.put("INVALID_TRUST_HASH2", "*");
+        replaceMap.put("INVALID_TRUST_HASH1", "0xef4073a0f2b305a38ec4050e4d3d28bc40ea63f5");
+        replaceMap.put("INVALID_TRUST_HASH2", "0xd2a4cff31913016155e38e474a2c06d08be276cf");
```
Comment (mialbu) [inline compiler/src/test/java/io/neow3j/compiler/PlaceholderSubstitutionTest.java:37]: Afaik, these are valid hashes, right? Then, can we rename the string to something else, e.g., `TRUST_HASH1`?

**Review by mialbu — APPROVED** [review]
Great tests! I only commented a minor thing about naming. 😼

File: compiler/src/test/java/io/neow3j/compiler/PlaceholderSubstitutionTest.java (hunk 1 above)
Comment (csmuller) [reply compiler/src/test/java/io/neow3j/compiler/PlaceholderSubstitutionTest.java:37]: thanks, i've missed that.

**Review by mialbu — APPROVED** [review]
💯

**Pull request merged**